components/canvas/webgl_thread.rs

```diff
@@ -331,16 +345,20 @@ impl WebGLThread {
                             // and in GLES. Rather than doing version detection, it does
                             // not hurt to enable them anyways.
 
-                            data.gl.enable(gl::POINT_SPRITE);
-                            let err = data.gl.get_error();
-                            if err != 0 {
-                                warn!("Error enabling GL point sprites: {}", err);
-                            }
-
-                            data.gl.enable(gl::PROGRAM_POINT_SIZE);
-                            let err = data.gl.get_error();
-                            if err != 0 {
-                                warn!("Error enabling GL program point size: {}", err);
+                            unsafe {
+                                // XXX: Do we even need to this?
```

Comment:
IDK, I also asked this on zulip: https://servo.zulipchat.com/#narrow/channel/263398-general/topic/GL.2FGLES.20versions.20supported, as this would allow us to remove some old GLES/GL handling.